package.json

```
@@ -32,14 +32,14 @@
   "homepage": "https://github.com/BigRoomStudios/strange-router#readme",
   "peerDependencies": {
     "prop-types": "15.x.x",
-    "react": "16.x.x",
+    "react": ">=16.x.x <18",
```

Not a major deal, but can we format this as we do elsewhere for ranges (like below for react-router, but also in strange-middle-end)?
```suggestion
    "react": ">=16 <18",
```